lib/backends/now/compute.rb

```diff
@@ -0,0 +1,640 @@
+module Backends
+  module Now
+    class Compute < Backends::Now::Base
```

Comment:
No dummies, please. If you need to use a dummy for testing, combine `dummy` backends for `compute` and `storage` with a real `network` backend for NOW.